elide-core/src/main/java/com/yahoo/elide/security/executors/ActivePermissionExecutor.java

```diff
@@ -268,11 +324,16 @@ public void executeCommitChecks() {
      *
      * @param expressions expressions to execute
      */
-    private void executeExpressions(final Expressions expressions,
+    private ExpressionResult executeExpressions(final Expressions expressions,
                                     final Class<? extends Annotation> annotationClass) {
         Expression expression = expressions.getOperationExpression();
         ExpressionResult result = expression.evaluate();
-        if (result == DEFERRED) {
+
+        // Record the check
+        Long checkOccurrences = requestScope.getCheckStats().getOrDefault(expression.toString(), 0L) + 1;
```

Comment:
Moving it(and the shortcircuit map) to `ActivePermissionExecutor` should also require the `PermissionExecutor` to have the getter declaration. I'm not sure if it makes sense for `BypassPermissionExecutor` to have these getters.